packages/react-hooks/src/recordUtils.ts

```diff
@@ -9,10 +9,11 @@ import type {
 import type { Constructor } from '@aries-framework/core/build/utils/mixins'
 
 import { RepositoryEventTypes } from '@aries-framework/core'
+import { useMemo } from 'react'
 import { map, filter, pipe } from 'rxjs'
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-type BaseRecordAny = BaseRecord<any, any, any>
+export type BaseRecordAny = BaseRecord<any, any, any>
```

Comment:
Could we maybe add a clarifying comment on what this `BaseRecordAny` is and what it does?

Reply:
Yes!